enzy_htp/chemical/atoms.py

```diff
@@ -9,7 +9,7 @@
 X_H_BOND_LENGTH: Dict[str, float] = {
         "C": 1.07,
         "N": 1.0
-}
+} # yapf: disable
```

Comment:
Get rid of this comment.

Reply:
Yapf will make it into one line if so. I want it to stay what it is rn. I searched about it and this is the solution they give https://github.com/google/yapf/issues/359